ui/panels-plugin/src/plugins/table/ColumnsEditor/ColumnEditorContainer.tsx

```diff
@@ -47,6 +52,14 @@ export function ColumnEditorContainer({
         gap={4}
       >
         <Stack direction="row" gap={1}>
+          <DragButton
```

Comment:
I'd rather see this button on the right side, to align with the drag&drop UX of panels